src/core/commandRunner/CommandRunner.ts

```diff
@@ -110,8 +110,8 @@ export default class CommandRunner {
       this.testCaseRecorder.commandErrorHook();
       const err = e as Error;
       vscode.window.showErrorMessage(err.message);
-      console.debug(err.message);
-      console.debug(err.stack);
+      this.graph.debug.log(err.message);
+      this.graph.debug.log(err.stack);
```

Comment:
I'm tempted to make these just `console.error`, because that way we can catch the stack trace the first time an error occurs.  WDYT?

Reply:
I was also thinking that.